### ISSUE TITLE
ONTAP: cDOT Dashboard - Ignore Node SVMs and vol0 volums

### ISSUE THREAD

**Issue by Falcon667** [NetApp/harvest#1533]
**Is your feature request related to a problem? Please describe.**
feature request. In the new cDOT dashboard there are also node SVM and vol0 volumes visible in the capacity dashboards.

**Describe the solution you'd like**
Ignore node SVMs and vol0 volumes
![image](https://user-images.githubusercontent.com/99657787/203261551-0eee6cbc-28ff-407a-9528-2eb8fd67d656.png)

**Additional context**
This would make the dashboard cleaner

**Comment by Hardikl** [comment]
@Falcon667 
- When you say excluding node SVMs from the svm capacity panels, Do you also prefer to exclude the admin SVMs from the same panel?

-  When you say excluding the vol0 volumes(which is node root volumes) from the volume capacity panels, Do you also prefer to exclude the vserver root volumes from the same panel?

**Comment by MrObvious** [comment]
I think this should be optional. There can be corner cases where this is useful.

**Comment by Falcon667** [comment]
@Hardikl I agree with @MrObvious in some corner cases it would make sense. In our company I can not remember me, that we ever used the capacity metric from the admin/node SVMs or node root volume.
